mapbox_vector_tile/encoder.py

```diff
@@ -56,13 +56,28 @@ def transform(shape, func):
     raise ValueError('Unknown geometry type, "%s"' % shape.type)
 
 
+def on_invalid_geometry_raise(shape):
+    raise Exception('Invalid geometry: %s' % shape.wkt)
```

Comment:
Do we want to raise plain `Exception`, or a more specific class such as `ValueError` or `RuntimeError` (or even a class of our own)?